tests/src/Unit/GroupTypeManagerTest.php

```diff
@@ -204,7 +204,7 @@ public function providerTestIsGroup() {
   /**
    * Tests getting all the groups of an entity type.
    *
-   * @covers ::getGroupsForEntityType
+   * @covers ::getGroupBundleIdsByEntityType
    */
   public function testGetGroupsForEntityType() {
```

Comment:
We can now also rename the test to `testGetGroupBundleIdsByEntityType()`. The documentation of this test can also be updated, it says that we are "getting all the groups" but we are in fact getting all the group bundle IDs.